### ISSUE TITLE
Data Highway Economic Variables

### ISSUE THREAD

**Issue by sheenhx** [DataHighway-DHX/whitepaper#6]
The initial economic variables shown in the table below were decided upon through optioneering and may be configured:

![](https://i.imgur.com/tK4d8Lr.png)

Table 1: Data Highway Economic Variables

Can you put this into a normal table using markdown?

**Comment by sheenhx** [comment]
Block Authoring / Production Rewards
The initial token issuance halving inflation strategy cycle (halving of the block production reward) shall occur every four (4) years using the "Decreasing-Supply Algorithm".

Also here and some other tables

**Comment by sheenhx** [comment]
Maybe some tables can't be done in this Markdown way?

**Comment by ltfschoen** [comment]
Yes we can generate markdown version of the table. But if anyone decides to change one of the values in the markdown version of the whitepaper, it won't automatically update all the other tables in the whitepaper, and we'd have to cascade those changes to the Google Spreadsheet where the calculations actually work too.

Since we want these to be configurable by the DAO, it may be best to generating a script that gets the latest values that the DAO uses on-chain, and generate a DataHighway front-end that specifically shows these tables and some visualizations
